docs/guides/update-guide/connectors/060-to-070.md

```diff
@@ -0,0 +1,18 @@
+---
+id: 060-to-070
+title: Update 0.6 to 0.7
+description: "Review which adjustments must be made to migrate from Connector SDK 0.6.x to 0.7.0."
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+With the Connector SDK [0.7.0 release](https://github.com/camunda/connector-sdk/releases/tag/0.7.0), we made breaking changes to the inbound
```

Comment:
As this is a breaking change, do we need to note this within the `http-webhook` and `github-webhook` documentation as well? Applies to `8.1` change as well.

Reply:
Makes sense - thank you for pointing it out